### PR TITLE
"Bricks" and "Brick Status" panels on Host and Volume dashboards doesn't use the same definition of brick state 

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/monitoring-integration#545
bugzilla: 1614000

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @cloudbehl please review

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json [hunk 1]
```diff
@@ -164,7 +164,7 @@
                     "value": "current"
                 }],
                 "datasource": "",
-                "description": "The Brick Status panel displays the status code of each brick for a given host.  \n0 = Started  \n8 = Stopped",
+                "description": "The Brick Status panel displays the status code of each brick for a given host.  \n0 = Up  \n8 = Down",
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: yes yes that is correct i will change this

**Comment by GowthamShanmugam** [comment]
@cloudbehl changes done

**Comment by cloudbehl** [comment]
@GowthamShanmugam can you please have a look at doc on which ju commented. We need to check with gluster people what should we use for brick status.

**Comment by shtripat** [comment]
Tag somebody from gluster team on BZ directly and ask.
As far as I know brick status should be `Up` and `Down` but lets confirm.

**Comment by GowthamShanmugam** [comment]
@julienlim @mbukatov need your comment here

**Comment by cloudbehl** [comment]
@shtripat if the status is Up and Down then we need to change the status panel of brick dashboard to say Up and Down. Right now it is Started and Stopped. We need to be consistent. 

@GowthamShanmugam ^^

**Comment by GowthamShanmugam** [comment]
volume_dashboard_brick_status
host_dashboard_brick_status

@mbukatov @julienlim

**Comment by GowthamShanmugam** [comment]
@cloudbehl

**Pull request merged**